synapse/lib/cli.py

```diff
@@ -436,6 +436,11 @@ async def runCmdLine(self, line):
         except asyncio.CancelledError:
             self.printf('Cmd cancelled')
 
+        except s_exc.ParserExit as e:
+            mesg = e.errinfo.get('mesg')
+            if mesg is not None:
+                self.printf(mesg)
```

Comment:
The parser already prints out the message before raising `ParserExit`, this appears to be causing it to print a second time:
```storm> !pullfile
usage: !pullfile [-h] sha256 filepath

!pullfile: error: the following arguments are required: sha256, filepath

!pullfile: error: the following arguments are required: sha256, filepath

storm>